app/controller/grid/ItemDeleterGridControllerMixin.js

```diff
@@ -0,0 +1,45 @@
+Ext.define('CpsiMapview.controller.grid.ItemDeleterGridControllerMixin', {
+    extend: 'Ext.Mixin',
+
+    onDeleteRowClick: function (tableView, rowIndex, colIndex, item, e, record, tableRow) {
+        var me = this;
+        // what to do for deleting
+        var doDelete = function () {
+            record.erase({
+                failure: function (/*record, operation*/) {
+                    // do something if the delete failed
+                },
+                success: function (/*record, operation*/) {
+                    // do something if the delete succeeded
+                },
+                callback: function (/*record, operation, success*/) {
+                    // do something whether the delete succeeded or failed
+                }
+            });
+        };
+        // what to do for not deleting
+        var dontDelete = function () {
```

Comment:
How is this configured? Shouldn't it be a property on the mixin at least? Also can use the https://docs.sencha.com/extjs/6.7.0/classic/Ext.html#property-emptyFn if required.